packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,13 @@
+{
+  "xdsl_meeting_slots": "Prise de rendez-vous",
+  "xdsl_meeting_error": "Une erreur est survenue lors la récupération des dates de rendez-vous disponibles : {{error}}",
+  "xdsl_meeting_description": "L'intervention d'un technicien est nécessaire afin de finaliser l'installation de votre ligne.",
+  "xdsl_meeting_fake_meeting": "Aucun des rendez-vous ci-dessous ne me convient, je veux être contacté par un technicien pour convenir d'un autre créneau horaire.",
+  "xdsl_meeting_no_fake_meeting": "Je choisis un horaire de rendez-vous.",
```

Comment:
Don't find where `xdsl_meeting_fake_meeting` and `xdsl_meeting_no_fake_meeting` are used ?